### PR TITLE
RHI contour example 2

### PR DESCRIPTION
**Pull request by Rumpkie**
This is the differential reflectivity example that Scott had discussed, I increased the opacity of reflectance map but not removed it because I wanted the example to cover how to add and adjust cmap alpha.

### CODE REVIEW

File: examples/plotting/plot_rhi_contours_differential_reflectivity.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+"""
+print __doc__
+
+# Author: Cory Weber (cweber@anl.gov)
+# License: BSD 3 clause
+import matplotlib.pyplot as plt
+import pyart
+import numpy as np
+import scipy.ndimage as spyi
```
Comment (jjhelmus) [inline examples/plotting/plot_rhi_contours_differential_reflectivity.py]: The recommended name to import this is as ndimage

**Comment by jjhelmus** [comment]
If you can rebase this, it should be good to go.

**Comment by Rumpkie** [comment]
I rebased it and made the style change you recommended

**Comment by jjhelmus** [comment]
@Rumpkie Looks good, the failing Travis-CI is unrelated.  Can you share a copy of the radar file (220629.mdv) or point out where it can be downloaded?

**Comment by jjhelmus** [comment]
Found this file, merging.

**Pull request merged**